frontend/src/metabase/query_builder/components/expressions/Editor/Editor.tsx

```diff
@@ -4,48 +4,50 @@ import cx from "classnames";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { useMount } from "react-use";
 import { t } from "ttag";
+import _ from "underscore";
 
 import { useSelector } from "metabase/lib/redux";
 import { getMetadata } from "metabase/selectors/metadata";
-import { Box } from "metabase/ui";
+import { Box, Button, Flex, Icon } from "metabase/ui";
 import * as Lib from "metabase-lib";
-import { format, isExpression } from "metabase-lib/v1/expressions";
-import type { Shortcut } from "metabase-lib/v1/expressions/complete";
+import { format } from "metabase-lib/v1/expressions";
```

Comment:
I typed a single character and I'm immediately punished with a red error message.
Previously the errors were shown only after bluring the editor.

![image](https://github.com/user-attachments/assets/6468cc8d-20ab-49c9-a33c-417d01de25b5)

Reply:
@mngr, @AlessioLaiso this veers a bit from the design goals of the project, but how would you feel if we only show errors on blur?